scripts/jenkins/clang.groovy

```diff
@@ -1,26 +1,35 @@
-configure = load 'scripts/jenkins/lib/configure.groovy'
-build     = load 'scripts/jenkins/lib/build.groovy'
-post      = load 'scripts/jenkins/lib/post.groovy'
-
 node('docker') {
+    def configure = load 'scripts/jenkins/lib/configure.groovy'
+    def build     = load 'scripts/jenkins/lib/build.groovy'
+    def post      = load 'scripts/jenkins/lib/post.groovy'
+
     def defaultDockerArgs = '-v /home/jenkins/.ccache:/usr/src/.ccache'
+    def defaultCMakeOptions =
+        '-DOGS_LIB_BOOST=System ' +
+        '-DOGS_LIB_VTK=System ' +
+        '-DOGS_ADDRESS_SANITIZER=ON ' +
+        '-DOGS_UNDEFINED_BEHAVIOR_SANITIZER=ON'
```

Comment:
Please add utils also to the build; there are specific ctests running if utils are enabled.